src/Main.elm

```diff
@@ -18,12 +20,13 @@ main =
 
 
 type alias Model =
-    {}
+    { language : Language
+    }
 
 
 init : () -> ( Model, Cmd Msg )
 init _ =
-    ( {}
+    ( (Model English)
```

Comment:
This is wrong I think. And you may also get some conflicts when you merge in the route code.